boot/boot.js

```diff
@@ -287,6 +287,136 @@ $tw.utils.decodeURIComponentSafe = function(s) {
 	return v;
 };
 
+/*************
+Code to make the permalink/permaview URLs more readable.
+(These comments are just to help reviewers; they will eventually be removed.)
+***************/
+
+
+/*************
+Helpers for encoding/decoding URIs.  **Note** that these are throw-away, only
+used to temporarily build `charMatch`, `spaceMatch`, `sentenceEnding`, and
+`sentenceTrailing`, all of which will eventually be hard-coded.
+***************/
+
+/*
+The character that will substitute for a space in the URL
+*/
+var SPACE = "_";
+
+/*
+The character added to the end to avoid ending with `.`, `?`, `!` or the like
+*/
+var TRAILER = '_';
+
+/*
+The character that will separate out the list elements in the URL
+*/
+var CONJUNCTION = ";";
```

Comment:
To save a bit of vertical space **for all 1-line** variable declarations below the var comments could be inline like so: 

```
// The character that will separate out the list elements in the URL
var CONJUNCTION = ";";

// Those of the allowed url characters claimed by TW
var CLAIMED = [SPACE, ":", CONJUNCTION];
```

Reply:
Ok.  The multiline version just looked like house style... but I didn't look very hard.